### ISSUE TITLE
Powershell stops initializing ssh-agent

### ISSUE THREAD

**Issue by nigurr** [dahlbyk/posh-git#292]
When I try to open new windows, I am getting this message and it doesn't show up the console prompt.
This apparently started happening after installation of `Win32 ssh`
https://github.com/PowerShell/Win32-OpenSSH

```
Windows PowerShell
Copyright (C) 2015 Microsoft Corporation. All rights reserved.

agent_start pid:30344, dbg:1, child:0, pipe:0
```

Can you please let me know the issue? Powershell opens up if I comment out `poshgit` call in `$profile`

**Comment by Abdullah2993** [comment]
I also have the same issue. Any thoughts?

**Comment by nigurr** [comment]
Temp Fix: https://github.com/dahlbyk/posh-git/pull/295

I hard coded GIT ssh path, change it according to your system. Looking for a way to set globally.

**Comment by dahlbyk** [comment]
See https://github.com/dahlbyk/posh-git/issues/258 for more information.

**Comment by dahlbyk** [comment]
Closing as duplicate of #258.

**Comment by dahlbyk** [comment]
This should be resolved by #586 in an upcoming v0.8 release.
